test3.js

```diff
@@ -0,0 +1,26 @@
+var foo = '' ;
+
+if(foo  === "bar") {
+  console.log( 'hey' )
+}
+
+
+var foo77 = '' ;
```

Comment:
Codacy has a fix for the issue: [Replace `''·` with `""`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5796)
```suggestion
var foo77 = "";
```